api/nakadi-event-bus-api.yaml

```diff
@@ -843,6 +864,9 @@ definitions:
               from the header into the metadata by Nakadi to avoid clients having to explicitly copy this.
             type: string
             example: 'JAh6xH4OQhCJ9PutIV_RYw'
+        required:
+          - eid
+          - occurred_at
```

Comment:
@jkliff maybe not every user would be interested in providing `occurred_at` or `eid`.